build/config/BUILDCONFIG.gn

```diff
@@ -275,11 +275,13 @@ if (target_os == "android") {
 # If a custom toolchain has been set in the args, set it as default. Otherwise,
 # set the default toolchain for the platform (if any).
 if (custom_toolchain != "") {
-  set_default_toolchain(custom_toolchain)
+  cobalt_toolchain = custom_toolchain
```

Comment:
Small nitpick but i'd leave the original code unchanged, and just add lines here to assign cobalt toolchain values. It's cleaner to understand i think, even if its more duplicative.

Also i supposed we can't `if(chrobalt)` anything in this file, because that condition / flag wouldn't have been evaluated yet ?